### ISSUE TITLE
Changing the "main" domain

### ISSUE THREAD

**Issue by mtimyself** [nextcloud/server#10587]
We installed our NC 13 instance under "domain A" because we had access to the DNS configuration at the time.
After installing and configuring the system we made it available under "domain B", "domain C", etc. as well.
Now we intend to have the system send mails from "domain B", which is easily configurable through the admin panel and works as intended but we wish the links that are generated, e.g. for account creation, to be from "domain C". Currently these links are still generated from "domain A" and we do not seem to have any directive pointing towards that in our config.php.
E: I have also tried setting the desired domain as the first trusted domain (array # 1 after IP on # 0) with no effect.

Long story short:
How do we change the domain that is used to generate links in the mails sent by NC (content, not sender)?

### Server configuration

**Operating system**:
Ubuntu 16.04.5 LTS

**Web server:**
nginx

**Database:**
MySQL

**PHP version:**
PHP 7.0.30-0ubuntu0.16.04.1 (cli) ( NTS )

**Nextcloud version:** (see Nextcloud admin page)
Nextcloud 13.0.5

**Updated from an older Nextcloud/ownCloud or fresh install:**
Fresh

**Where did you install Nextcloud from:**
Official package

**Signing status:**
<summary>No errors have been found.</summary>

**Comment by MorrisJobke** [comment]
Have a look at the "overwrite..." parameters: https://docs.nextcloud.com/server/14/admin_manual/configuration_server/config_sample_php_parameters.html#proxy-configurations

As this seems to be a setup issue I would like to ask you to raise your question in the forums: https://help.nextcloud.com

If you wish support with setup issues from Nextcloud GmbH we offer this as part of the Nextcloud subscription. Learn more about this at https://nextcloud.com/enterprise/
